src/cljs/ui/analysis_results/page.cljs

```diff
@@ -59,7 +60,7 @@
             [:div.export
              [:span "Support values"]
              [button {:text "Export To CSV"
-                      :on-click #()
+                      :on-click #(prn "TODO")
```

Comment:
Just in case you forgot about it

Reply:
it's part of this ticket: https://github.com/fbielejec/spread/issues/129